src/pages/dashboard/dashboard.tsx

```diff
@@ -59,80 +77,83 @@ const Dashboard = () => {
-                  </p>
-                </div>
-              </>
-            }
-            footer={
-              <Button type="link" onClick={() => setOpenModal('withdraw')} disabled={!canWithdraw}>
-                Withdraw
-              </Button>
-            }
-          />
-        </div>
-        <div className={styles.stakingBoxWrap} data-cy="staking-panel">
-          <BorderedBox
-            header={
-              <Header>
-                <h5>Staking</h5>
-                <div>
-                  <Button onClick={() => setOpenModal('stake')} disabled={!canStake}>
-                    + Stake
-                  </Button>
-                  <Tooltip overlay="You need to deposit API3 tokens before staking">
-                    <img src={images.help} alt="new proposal help" className={globalStyles.helpIcon} />
-                  </Tooltip>
-                </div>
-              </Header>
-            }
-            content={
-              <>
-                <div className={classNames(globalStyles.textCenter, globalStyles.mbLg)}>
-                  <p className={styles.borderedBoxContentTitle}>staked</p>
-                  <p className={globalStyles.textXLarge} data-cy="staked">
-                    {data ? formatAndRoundApi3(data.userStaked) : UNKNOWN_NUMBER}
-                  </p>
-                </div>
-                <div className={globalStyles.textCenter}>
-                  <p className={styles.borderedBoxContentTitle}>unstaked</p>
-                  <p className={globalStyles.textXLarge} data-cy="unstaked">
-                    {data ? formatAndRoundApi3(data.userUnstaked) : UNKNOWN_NUMBER}
-                  </p>
-                </div>
-              </>
-            }
-            footer={
-              // TODO: In case there is a pending unstake there should be no button, just green arrow (see figma)
-              <Button type="link" onClick={() => setOpenModal('unstake')} disabled={!canInitiateUnstake}>
+              </div>
+            </CardHeader>
+          }
+          content={
+            <div className={styles.cardContent}>
+              <div>
+                <p className={styles.cardContentTitle}>total</p>
+                <p className={styles.cardContentValue} data-cy="balance">
+                  {tokenBalances ? formatAndRoundApi3(tokenBalances.userTotal) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+              <div>
+                <p className={styles.cardContentTitle}>withdrawable</p>
+                <p className={styles.cardContentValue} data-cy="withdrawable">
+                  {tokenBalances ? formatAndRoundApi3(tokenBalances.withdrawable) : UNKNOWN_NUMBER}
+                </p>
+              </div>
+            </div>
+          }
+          contentFooter={
+            <Button type="secondary" size="sm" onClick={() => setOpenModal('withdraw')} disabled={!canWithdraw}>
+              Withdraw
+            </Button>
+          }
+        />
+
+        <Card
```

Comment:
The Card should have dark border when the Unstake was initiated

![image](https://github.com/user-attachments/assets/c40b432d-0f5a-4107-9ee5-a9ea17125497)